internal/runbits/requirements/requirements.go

```diff
@@ -209,6 +222,61 @@ func (r *RequirementOperation) ExecuteRequirementOperation(
 		pg = nil
 	}
 
+	if r.Auth.Authenticated() && operation == bpModel.OperationAdded && ns.Type() == model.NamespacePackage {
+		pg = output.StartSpinner(out, locale.Tr("progress_cve_search", requirementName), constants.TerminalAnimationInterval)
+
+		vulnerabilities, err := model.FetchVulnerabilitiesForIngredient(r.Auth, &request.Ingredient{
+			Namespace: ns.String(),
+			Name:      requirementName,
+			Version:   requirementVersion,
+		})
+		if err != nil {
+			return errs.Wrap(err, "Failed to retrieve vulnerabilities")
```

Comment:
`pg` is not stopped on error here.

Reply:
There is a defer at the top of this function that should handle that.